lib/moon/design/button.ex

```diff
@@ -69,6 +70,7 @@ defmodule Moon.Design.Button do
         disabled: @disabled,
         type: @type,
         form: @form,
+        close: @close,
```

Comment:
what is it about?

Reply:
Probably my mistake, something additional that is not needed